Files/Helpers/StorageItemHelpers.cs

```diff
@@ -20,35 +22,83 @@ public static async Task<IStorageItem> ToStorageItem(this IStorageItemWithPath i
             FilesystemResult<StorageFile> file = null;
             FilesystemResult<StorageFolder> folder = null;
 
-            if (associatedInstance == null)
+            if (path.EndsWith(".lnk") || path.EndsWith(".url"))
```

Comment:
Should there be some kind of ignore case here?

Reply:
What do you mean?

Reply:
There is this check to skip .lnk .url files. See `TODO` two lines below for explanation. @yaichenbaum

Reply:
And this https://github.com/files-community/Files/pull/4346#discussion_r602943761

Reply:
What if the extension is `.Url`? Sidenote, please don't resolve active conversations until I have chance to respond 🙂.